src/CMakeLists.txt

```diff
@@ -46,6 +47,7 @@ endif()
 
 add_dependencies(RadiumLibs ${RADIUM_COMPONENTS})
 set(RADIUM_COMPONENTS ${RADIUM_COMPONENTS} PARENT_SCOPE)
+set(RADIUM_MISSING_COMPONENTS ${RADIUM_MISSING_COMPONENTS} PARENT_SCOPE)
```

Comment:
Why is the variable `RADIUM_MISSING_COMPONENTS` set twice in the same file ?

Reply:
Right, it's uneeded.